src/firefly/js/visualize/ui/TableSelectViewPanel.jsx

```diff
@@ -49,9 +54,34 @@ export class TapSearchPanel extends PureComponent {
     render() {
         const {serviceUrl} = this.state;
 
+        // keep all tabs the same size
+        const tabWrapper = {padding:5, minWidth:1100, minHeight:100};
+
+        const placeholder = serviceUrl ? `Using <${serviceUrl}>. Replace...` : 'Select TAP...';
+
         return (
             <div style={{padding: 10}}>
-                <ListBoxInputFieldView
+                <div style={{paddingBottom:5}}>
+                    <CreatableSelect
+                        options={TAP_SERVICE_OPTIONS}
+                        isClearable={true}
+                        onChange={(selectedOption) => {
```

Comment:
I think long function inside the JSX make it hard to read.  If you need a closure function you declare it above or just make a static function that you use a closure to pass all the parameters to.